File: src/main/java/org/hotwheel/stock/model/StockSubscribe.java
package org.hotwheel.stock.model;

import java.util.Date;

/**
 * 股票订阅
 *
 * Created by wangfeng on 2017/3/14.
 * @version 1.0.0.
 */
public class StockSubscribe {
    private String flag;
    private String phone;
    private String code;
    private Date createTime;
    private Date sendDate;
    private String operator;
    private long id;

    public String getFlag() {
        return flag;
    }

    public void setFlag(String flag) {
        this.flag = flag;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public Date getSendDate() {
        return sendDate;
    }

    public void setSendDate(Date sendDate) {
        this.sendDate = sendDate;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }
}

File: store/models/stock/stock_subscribe.go
package stock

import (
	"time"
)

type StockSubscribe struct {
	Flag       string    `xorm:"default '00' comment('标志($): 00-禁止订阅,01-正常订阅') CHAR(2)"`
	Phone      string    `xorm:"not null comment('客户ID') index CHAR(32)"`
	Code       string    `xorm:"not null comment('股票代码') index CHAR(32)"`
	Createtime time.Time `xorm:"comment('创建时间($)') DATETIME"`
	Senddate   time.Time `xorm:"comment('发送日期') DATE"`
	Remark     string    `xorm:"comment('策略命中备注') TEXT"`
	Operator   string    `xorm:"default 'system' comment('操作人(?$)') VARCHAR(50)"`
	Id         int       `xorm:"not null pk autoincr INT(10)"`
}
